src/bindings/python/src/openvino/runtime/utils/broadcasting/__init__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
I think we should raise a warning about its deprecation since 2025 and our intention to remove in 2026 release.
It relates all `runtime` modules